### PR TITLE
Export dwg_add_ATTDEF function

### PR DESCRIPTION
**Pull request by michal-josef-spacek**
attdef is regular entity, independent on other

### CODE REVIEW

**Review by rurban — CHANGES_REQUESTED** [review]
See dwg_api.h on dwg_add_Attribute:

  This adds the ATTRIB and ENDBLK to the insert,
  and the ATTDEF and ENDBLK to the block, if missing.

Letting the user all do this will be too complicated.

File: examples/dwgadd.example [hunk 1]
```diff
@@ -4,8 +4,8 @@
 #readjson "example_2000.json"
 #version 2000
 text "test" (0.0 1.0 0.0) 8
-attribute 8 0 "prompt" (0.0 1.0 0.0) "tag" "text"
```
[the remark this answers is not in the document]
Comment (michal-josef-spacek) [reply examples/dwgadd.example:7]: Yes, we need to add an attrib.

The question is, does dwg_add_Attribute make sense?

[the remark this answers is not in the document]
Comment (michal-josef-spacek) [reply examples/dwgadd.example:7]: Hm, I need to look more.
In AutoCAD script is an attdef and insert (with values for attrib based on attdef)

**Comment by rurban** [comment]
Merged with various fixups

**Pull request closed without merging**